e2e/jest.test.js

```diff
@@ -13,6 +13,14 @@ describe('examples/jest', () => {
     BUILDKITE_ANALYTICS_DEBUG_ENABLED: "true"
   }
 
+  test('it outputs a warning when --forceExit option is used', (done) => {
+    exec('jest --forceExit', { cwd, env }, (error, stdout, stderr) => {
+      expect(stderr).toMatch(/--forceExit could potentially terminate any ongoing processes that are attempting to send test executions to Buildkite./);
+
+      done()
+    }, 1000) // 1s timeout
```

Comment:
Why is this timeout necessary?

Reply:
I think the original intention was to support a timeout, but the argument is in the wrong place.

I copied the structure of the two tests below. 

Looks like this is the exec function
https://nodejs.org/api/child_process.html#child_processexeccommand-options-callback

It has this method signature

`child_process.exec(command[, options][, callback]) `

It doesn't support a fourth argument, so it seems the 1000 timeout is redundant.

However, I noticed the `test()` function supports a timeout 

https://jestjs.io/docs/api#testname-fn-timeout

Reply:
I've added a new commit to correct the other places where the timeout argument is in the wrong place. 6e9e2f1

Reply:
@AeroCross I've merged this PR on Jasmine's approval. But let me know if you have any further concerns about this!